grails-app/assets/vendor/jquery.timeentry.package-2.0.1/jquery.plugin.js

```diff
@@ -0,0 +1,344 @@
+/* Simple JavaScript Inheritance
```

Comment:
Is there a reason this couldn't go in the ecodata-client-plugin?
If it needs to be in MERIT, please use build.gradle if possible as it's much easier to manage library versions that way.